From 20fe93531a92149c3e2810d80fcc2a5e73459b56 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Tue, 5 Nov 2024 17:58:46 +0100
Subject: [PATCH 1345/1695] dts: stm32: fix RTC GIC interrupt ID in
 stm32mp211.dtsi

Correct GIC interrupt lines for RTC peripherals that is wrong.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git feeb1246e1c56f55850c783d64530cc9e4867e47 dts: stm32: fix RTC GIC interrupt ID in stm32mp211.dtsi]
Change-Id: Ib3d626370475bfda775984fac561e8c3159c819c
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/416638
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 665eba4bd..863f94dce 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -2716,7 +2716,7 @@
 			reg = <0x46000000 0x400>;
 			clocks = <&rcc CK_BUS_RTC>, <&rcc RTC_CK>;
 			clock-names = "pclk", "rtc_ck";
-			interrupts = <GIC_SPI 15 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts = <GIC_SPI 13 IRQ_TYPE_LEVEL_HIGH>;
 		};
 
 		tamp: tamp@46010000 {
-- 
2.47.2

